app/styles/headers/_fourth.scss

```diff
@@ -0,0 +1,65 @@
+.header__fourth {
+    max-width: 1180px;
+    width: 100%;
+    height: 100vh;
```

Comment:
please, don't use `100vh` for header height. It's always better when height of block depends on its content.